ci-operator/step-registry/cucushift/upgrade/vsphere-hw/cucushift-upgrade-vsphere-hw-commands.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ -f "${SHARED_DIR}/kubeconfig" ]] ; then
+    export KUBECONFIG=${SHARED_DIR}/kubeconfig
+fi
+
+# Setup proxy if it's present in the shared dir
+if [[ -f "${SHARED_DIR}/proxy-conf.sh" ]] 
+then
+    # shellcheck disable=SC1091
+    source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+source "${SHARED_DIR}/govc.sh"
+# list all the virtual machines in the folder/rp
+cluster_name=$(<"${SHARED_DIR}"/clustername.txt)
+vm_path="/${GOVC_DATACENTER}/vm/${cluster_name}"
+clustervms=$(govc ls "${vm_path}-*")
+
+first_vm=$(echo "${clustervms}" | cut -d" " -f1)
```

Comment:
this command seems unable to get first vm, because var `${clustervms}` contains several lines, each node one line.
```
$ echo "${clustervms}" | cut -d" " -f1
/SDDC-Datacenter/vm/qeci-40980-rm45q/control-plane-1
/SDDC-Datacenter/vm/qeci-40980-rm45q/control-plane-0
/SDDC-Datacenter/vm/qeci-40980-rm45q/compute-1
/SDDC-Datacenter/vm/qeci-40980-rm45q/compute-0
/SDDC-Datacenter/vm/qeci-40980-rm45q/lb-0
/SDDC-Datacenter/vm/qeci-40980-rm45q/control-plane-2
```